cvasl/vendor/comscan/neurocombat.py

```diff
@@ -10,25 +10,31 @@
 
 import numpy as np
 import pandas as pd
-from neuroCombat.neuroCombat import standardize_across_features
-from neuroCombat.neuroCombat import find_parametric_adjustments
-from neuroCombat.neuroCombat import fit_LS_model_and_find_priors
-from neuroCombat.neuroCombat import find_non_eb_adjustments
-from neuroCombat.neuroCombat import find_non_parametric_adjustments
+from ..neurocombat.neurocombat.neuroCombat import standardize_across_features
+from ..neurocombat.neurocombat.neuroCombat import find_parametric_adjustments
+from ..neurocombat.neurocombat.neuroCombat import fit_LS_model_and_find_priors
+from ..neurocombat.neurocombat.neuroCombat import find_non_eb_adjustments
+from ..neurocombat.neurocombat.neuroCombat import find_non_parametric_adjustments
```

Comment:
```suggestion
from ..neurocombat.neurocombat import standardize_across_features
from ..neurocombat.neurocombat import find_parametric_adjustments
from ..neurocombat.neurocombat import fit_LS_model_and_find_priors
from ..neurocombat.neurocombat import find_non_eb_adjustments
from ..neurocombat.neurocombat import find_non_parametric_adjustments
```

I think this should fix (at least one of) your Sphinx build issue.